lib/vicadmin/validate.go

```diff
@@ -172,20 +174,42 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 
 	// Network Connection Check
 	hosts := []string{
-		"google.com:80",
-		"docker.io:443",
+		"http://google.com",
+		"https://docker.io",
 	}
+
 	nwErrors := []error{}
 
-	for _, host := range hosts {
-		conn, err := net.DialTimeout("tcp", host, time.Minute/2)
+	// create a http client with a custom transport. The transport uses the [s/h]proxy defined in vicmachine create if applicable.
+	client := &http.Client{Timeout: 30 * time.Second}
+	vchArgs := make(map[string]string)
+	for _, args := range vch.VicMachineCreateOptions {
```

Comment:
vicmachine create options is added for debug purpose so far. Why not read from env settings?